defaults/main.yml

```diff
@@ -1,3 +1,5 @@
 ---
 remi_repo_url: "http://rpms.remirepo.net/enterprise/remi-release-{{ ansible_distribution_major_version }}.rpm"
 remi_repo_gpg_key_url: "http://rpms.remirepo.net/RPM-GPG-KEY-remi"
+remi_repofile_path: "/etc/yum.repos.d/remi-php70.repo"
```

Comment:
This default seems fairly specific, just for PHP 7...

Reply:
I think, it should use more general repofile, such `/etc/yum.repos.d/remi.repo`